src/bun.js/api/server.zig

```diff
@@ -510,6 +585,7 @@ pub const ServerConfig = struct {
                         result.cert = native_array;
                         result.cert_count = 1;
                         any = true;
+                        result.requires_custom_request_ctx = true;
```

Comment:
is this boolean necessary?

Reply:
was added to fix this: https://github.com/oven-sh/bun/pull/6621#discussion_r1367572919